crates/host/src/wasmbus/mod.rs

```diff
@@ -425,6 +445,38 @@ impl Lattice {
             .build()
             .context("failed to build runtime")?;
         let event_builder = EventBuilderV10::new().source(host_key.public_key());
+
+        let jetstream = async_nats::jetstream::new(nats.clone());
+        // TODO: Use prefix
+        let bucket = format!("LATTICEDATA_{prefix}", prefix = "default");
+        jetstream
+            .create_stream(async_nats::jetstream::stream::Config {
+                allow_direct: true,
+                allow_rollup: true,
+                deny_delete: true,
+                discard: async_nats::jetstream::stream::DiscardPolicy::New,
+                duplicate_window: Duration::from_nanos(120_000_000_000),
+                max_bytes: -1,
+                max_consumers: -1,
+                max_message_size: -1,
+                max_messages: -1,
+                max_messages_per_subject: 1,
+                name: format!("KV_{bucket}"),
+                num_replicas: 1,
+                subjects: vec![format!("$KV.{bucket}.>")],
+                ..async_nats::jetstream::stream::Config::default()
+            })
+            .await
+            .map_err(|e| anyhow!(e).context("failed to create data bucket"))?;
+        let data = jetstream
```

Comment:
Naming nit: `store` (or `kv_store`, or `lattice_data_store`) matches the underlying type and is more specific than `data`

Reply:
The key is "LATTICEDATA", `data` directly corresponds to the key. `store` is too opaque, `kv_store` does not convey any semantic meaning, `lattice_data_store` is redundant, since the struct *is* called `Lattice`

I'm happy to rename to `data_store` if you prefer